bundles/org.openhab.binding.haywardomnilogic/README.md

```diff
@@ -106,11 +149,13 @@ Hayward OmniLogic Connection Parameters:
 | Channel Type ID  | Item Type            | Description          | Read Write |
 |------------------|----------------------|----------------------|:----------:|
 | pumpEnable       | Switch               | Pump enable          |     R/W    |
-| pumpSpeedPercent | Number:Dimensionless | Pump speed in %      |     R/W    |
-| pumpSpeedRpm     | Number               | Pump speed in RPM    |     R/W    |
+| pumpSpeedPercent | Number:Dimensionless | Pump speed (%)       |     R/W    |
+| pumpSpeedRpm     | Number: Frequency    | Pump speed in rpm    |     R/W    |
 | pumpSpeedSelect  | String               | Pump speed presets   |     R/W    |
 | pumpState        | String               | Pump state           |      R     |
-| pumpLastSpeed    | Number:Dimensionless | Pump last speed in % |      R     |
+| pumpLastSpeed    | Number:Dimensionless | Pump last speed (%)  |      R     |
+
+**Item Types Number:Dimensionless should have the units (i.e. %) defined in the Unit metadata 
```

Comment:
Yes, that is true but this is not specific to this binding.
Having this sentence repeated 3 times in your README is maybe excessive. Maybe make it global and mentioned only one time.